### PR TITLE
Fix outdated tag

### PR DESCRIPTION
**Pull request by DanielW10001**
According to `examples/sample-allOptions.xml`, `<logmode>` has been replaced by `<log></log>`; so does `rotate`.

### CODE REVIEW

**Comment by DanielW10001** [comment]
> Thanks!

You are welcomed! Also thank you for you excellent work!

**Comment by NextTurn** [comment]
CI failed. Reopening to start a new build.

**Comment by NextTurn** [comment]
CI failed again... Something went wrong with Nuget. Not a blocker though.

**Comment by DanielW10001** [comment]
@NextTurn 
Just dig into the source and find more `<logmode>` and `<log mode="MODE">` are mixed up.
I would like to make a fix of this, but I need to know which one we want to support? `<logmode>`, `<log mode="MODE">` or both?

**Comment by NextTurn** [comment]
https://github.com/winsw/winsw/blob/1b8cbccd8a4b5ac843b9343e7901dfcf749d05b3/src/Core/WinSWCore/ServiceDescriptor.cs#L342-L343

`<log mode="MODE">` is preferred but we support both for backward compatibility.

**Comment by DanielW10001** [comment]
Should we replace `<logmode>` in docs and test string with `<log mode="MODE">`, then?

**Comment by oleg-nenashev** [comment]
I am recovering CI

**Comment by DanielW10001** [comment]
I made another commit to replace all `<logmode>rotate</logmode>` with `<log mode="roll"></log>`, and `rotate` with `roll`, while leaving deprecated `rotate` mode and its description untouched.

**Comment by oleg-nenashev** [comment]
@DanielW10001 could you please merge your branch with master?

**Comment by DanielW10001** [comment]
I would love to, but I am not authorized to merge pull request. I am afraid only owner can do that.

**Comment by NextTurn** [comment]
@oleg-nenashev has fixed the build pipeline. Reopening to trigger a new build.

**Comment by oleg-nenashev** [comment]
`The test source file "D:\a\1\s\src\Test\winswTests\bin\Release\netcoreapp3.1\winswTests.dll" provided was not found`

Caused by source code corruption:

```
  "C:\Users\VssAdministrator\.nuget\packages\ilmerge\3.0.29\tools\net452\ILMerge.exe" /targetplatform:v4.5 /out:"D:\a\1\s\artifacts\WinSW.NET461.exe" "bin\Release\net461\WindowsService.exe" "bin\Release\net461\WinSWCore.dll" "bin\Release\net461\SharedDirectoryMapper.dll" "bin\Release\net461\RunawayProcessKiller.dll" "bin\Release\net461\log4net.dll"
  SharedDirectoryMapper -> D:\a\1\s\src\Plugins\SharedDirectoryMapper\bin\Release\net20\SharedDirectoryMapper.dll
  winsw -> D:\a\1\s\src\Core\ServiceWrapper\bin\Release\netcoreapp3.1\WindowsService.dll
Extensions\RunawayProcessKillerTest.cs(31,14): error CS1002: ; expected [D:\a\1\s\src\Test\winswTests\winswTests.csproj]
Util\CLITestHelper.cs(23,14): error CS1002: ; expected [D:\a\1\s\src\Test\winswTests\winswTests.csproj]
Extensions\RunawayProcessKillerTest.cs(31,18): error CS1002: ; expected [D:\a\1\s\src\Test\winswTests\winswTests.csproj]
Util\CLITestHelper.cs(23,18): error CS1519: Invalid token '"></log>' in class, struct, or interface member declaration [D:\a\1\s\src\Test\winswTests\winswTests.csproj]
Extensions\RunawayProcessKillerTest.cs(31,18): error CS1010: Newline in constant [D:\a\1\s\src\Test\winswTests\winswTests
```

**Comment by DanielW10001** [comment]
I am lookint into it.

**Comment by DanielW10001** [comment]
The problem seem to be fixed.

File: src/Test/winswTests/ServiceDescriptorTests.cs [hunk 1]
```diff
@@ -57,8 +57,8 @@ public void IncorrectStartMode()
   <description>The service.</description>
   <executable>node.exe</executable>
   <arguments>My Arguments</arguments>
-  <startmode>rotate</startmode>
-  <logmode>rotate</logmode>
+  <startmode>roll</startmode>
```
Comment (NextTurn) [inline src/Test/winswTests/ServiceDescriptorTests.cs:60]: ... ;)

**Comment by oleg-nenashev** [comment]
Yes, thanks for the patch!

**Pull request merged**